api/controllers/CheckUploadedDocumentsController.js

```diff
@@ -17,6 +21,32 @@ const CheckUploadedDocumentsController = {
         });
     },
 
+    async _addUserDataToDB(req) {
+        const userModels = getUserModels(
+            req._sails.config.userServiceSequelize
+        );
+        const userDataFromDB = await userModels.User.findOne({
+            where: { email: req.session.email },
+        });
+        const accountDetailsFromDB = await userModels.AccountDetails.findOne({
+            where: { user_id: userDataFromDB.id },
+        });
+
+        const userDetails = {
+            application_id: req.session.appId,
+            first_name: accountDetailsFromDB.first_name,
+            last_name: accountDetailsFromDB.last_name,
+            telephone: accountDetailsFromDB.telephone,
+            mobileNo: accountDetailsFromDB.mobileNo,
+            email: userDataFromDB.email,
+            confirm_email: userDataFromDB.email,
+            has_email: true,
+        }
+        await UsersBasicDetails.create(userDetails);
```

Comment:
What happens if the create fails?  It maybe covered in the create function but it looks like it needs error handling here.